### ISSUE TITLE
Discover via Bluetooth?

### ISSUE THREAD

**Issue by flatsiedatsie** [home-assistant-libs/python-matter-server#188]
I received a real Matter device today (Tapo P125M mini), and I'm trying to pair it. Running on a Raspberry Pi 4, Bullseye with Python 3.9.

When I run the discover command I see a lot of `mdns` messages.. but nothing about doing a bluetooth scan. Each time the result is an empty array.

Then I found this in the [documentation TODO](https://github.com/orgs/home-assistant-libs/projects/1/views/1?pane=issue&itemId=19503917):

```
Mention commissioning with the server directly is disabled due to coexitence issues with bleutooth
```

Does that mean it's not possible to pair my device? Should I use the manufacturer's app to pair it so it becomes available on the network, and then pair it again? 

More importantly, is this a temporary situation? I really want to avoid having to install apps on phones.

Some other questions I have:
- Is there support for commissioning with the manual pairing code?
- If not, is it possible to translate a manual pairing code into an `MT:` code somehow?
- When providing the wifi credentials, do I have to give the plaintext password or can I use the 'encrypted password'? I'd rather not store the plaintext wifi password on the Raspberry Pi.
- I notice it's trying to discover on `eth0`, but this predictably fails since the Raspberry Pi is connected via WiFi, not Ethernet. I don't seen any debug messages about it scanning on WLAN0. Should I be seeing those?

```
DEBUG:chip.DIS:Warning: Attempt to mDNS broadcast failed on eth0`)
```

**Comment by balloob** [comment]
To pair a Matter device, open Home Assistant on your mobile phone, go to config -> devices & services -> find Matter card and click ‘Configure’ and click on “Commission with mobile app”.

Alternatively, if you’re on Android, it should automatically pop up a pairing screen for Matter devices and you can pick Home Assistant as an option.

**Comment by flatsiedatsie** [comment]
I don't use Home Assistant. I'm using this library in [another project](https://github.com/createcandle/matter-adapter).

```
The goal of this project is primary to have Matter support in Home Assistant but its universal approach makes it suitable to be used in other projects too.
```

**Comment by flatsiedatsie** [comment]
I noticed this when doing a pairing attempt, perhaps it sheds some light.

```
ERROR:chip.CTL:Commissioning discovery over BLE failed: ../src/platform/Linux/BLEManagerImpl.cpp:814: CHIP Error 0x00000032: Timeout
ERROR:chip.-:../src/platform/Linux/BLEManagerImpl.cpp:814: CHIP Error 0x00000032: Timeout at ../src/controller/SetUpCodePairer.cpp:299
```

**Comment by balloob** [comment]
This feature works in the server, as we use it for development in Home Assistant.

**Comment by flatsiedatsie** [comment]
Ok, thanks. Then I will need to figure out why it doesn't seem to be working. Perhaps it's a permisisons issue.

**Comment by flatsiedatsie** [comment]
I managed to get the Bluetooth scan to work! I now see lots of Bluetooth devices scrolling by.

I'm still working out what is different now. One thing I noticed is it seems to only work when the Tapo Mini plug has just been booted. Or perhaps the Tapo blocks further attempts if one fails.

The process still halts as described in [this issue](https://github.com/home-assistant-libs/python-matter-server/issues/190), so I'm not able to fully pair yet. But I'm one step closer.
